src/Components/Header/Header.js

```diff
@@ -1,12 +1,64 @@
 import React, { Component } from "react";
+import "./Header.scss";
 
 class Header extends Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
   render() {
-    return <></>;
+    return (
+      <>
+        <div className="header">
+          {/* user__Menu */}
+          <ul className="header__menu">
```

Comment:
우리는 이제 sass의 nesting 기능을 이용합니다. 
그래서 className을 작성하실 때, 굳이 부모 태그의 className을 함꼐 작성하면서 
태그 하나하나 이름을 분리해 주지 않아도, nesting 기능을 통해 css 속성이 분리될 수 있습니다. 
그래서 위와 같은 태그의 경우도 아래와 같이 간단하게 className을 작성해주실 수 있습니다. 

다른 className들도 nesting기능을 이용하니, 간단하게 변경해보세요! 

더불어 그런 의미에서 react에서는 BEM이나 snake case보다는 camelCase 네이밍이 많이 쓰입니다.

```suggestion
          <ul className="menu">
```

Reply:
JSX부분도 camelCase로 작성하는군요. 수정하겠습니다.